### DISCUSSION TITLE
Please add your favourite Anime.

### DISCUSSION THREAD

**Discussion by LikkaLiu** [NAIST-SE/AttractingContributors#1108] [General]
https://github.com/LikkaLiu/AttractingContributors

**Comment by gill876** [comment]
Mine is definitely Psycho-Pass!

Please consider making a contribution to my project!
https://github.com/NAIST-SE/AttractingContributors/discussions/1089#discussion-6846489

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git
